packages/devextreme/js/__internal/ui/chat/chat_message_group.ts

```diff
@@ -118,11 +123,29 @@ class MessageGroup extends Widget<MessageGroupOptions> {
     $messageGroupInformation.appendTo(this.element());
   }
 
+  _removePrevBubbleLastClass(): void {
```

Comment:
This name is unclear. Could we change it to something more understandable?

Reply:
I have used more common name `_updateLastBubbleClasses` here: [b9e970178359975c6f2be01c472fdf8b32dbdf8e]